.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -35,6 +35,8 @@ body:
       1. Implement the following server or a client '...'
       2. Then run '...'
       3. An error occurs.
+      The chances of someone looking at your issue are *vastly* improved if you provide
```

Comment:
Personally, I'd put it into the description of the field rather than the placeholder that disappears as soon as somebody clicks on it... check the UX here https://github.com/aio-libs/aiohttp/issues/new?assignees=aio-libs%2Ftriagers&labels=bug&projects=&template=bug_report.yml

Reply:
Feel free to adjust. I'm not sure how I'd format it into the description.